assets/css/edd-admin.css

```diff
@@ -2048,3 +2048,59 @@ td.edd_order_price {
 	font-style: italic;
 	width: 150px;
 }
+
+
+/* Promo Notices
+-------------------------------------------------------------- */
+.edd-admin-notice-top-of-page {
+	font-size: 15px;
+	line-height: 1.4;
+	color: #fff;
+	margin-left: -20px;
+	padding: 12px 32px 12px 20px;
+	background: #2d6ca2;
```

Comment:
I think we're going for consistent colours across all brands (so the same colour, rather than unique to the brand). One of the reasons being that we wanted to avoid ever using a red that could imply an error/warning. Sean and Spencer worked up the CSS and we're meant to all copy the same thing across.